extensions/ql-vscode/src/remote-queries/remote-queries-markdown-generation.ts

```diff
@@ -144,22 +144,67 @@ function highlightCodeLines(
   return `${partiallyHighlightedLine.plainSection1}<strong>${partiallyHighlightedLine.highlightedSection}</strong>${partiallyHighlightedLine.plainSection2}`;
 }
 
-function buildMarkdownAlertMessage(interpretedResult: AnalysisAlert): string {
+function buildMarkdownAlertMessage(
+  interpretedResult: AnalysisAlert,
+  language: string
+): string[] {
+  const hasPathResults = interpretedResult.codeFlows.length > 0;
+  if (hasPathResults) {
+    // For path-problem queries, the "alert message" is an expandable section containing the path results.
+    return buildMarkdownPathResults(interpretedResult, language);
```

Comment:
🤔 I'm a little surprised by this - do we not want to expand message tokens for path-problem queries?

Reply:
This is another limitation of markdown 😢 As far as I'm aware, you can't have a clickable link (e.g. to an alert location) in the title of an expandable section, so it has to be plain-text for the path-problem case 🤷🏽 

(Maybe we can play around with the design a bit more though - not sure what the best "look" is 😅 )

Reply:
Oh interesting, thanks for the explanation! I think the current approach could lead to messages looking a bit weird (containing `[1]` etc.) so perhaps we should come up with some other "UX". How about having a "Show paths" section that is expandable then instead of using the message? A bit like we do with UI rendering of results.

Reply:
Done!